Cargo.toml

```diff
@@ -81,3 +81,7 @@ debug = 1
 [patch.crates-io]
 # Waiting on https://github.com/hyperium/headers/pull/99.
 headers = { git = "https://github.com/MaterializeInc/headers.git" }
+# Until https://github.com/jorgecarleitao/parquet-format-rs/pull/2 is merged and released
+parquet-format-async-temp = { git = "https://github.com/danhhz/parquet-format-rs", branch = "dep" }
+# Until https://github.com/prisma/tiberius/pull/189 is released
+tiberius = { git = "https://github.com/prisma/tiberius", branch = "main" }
```

Comment:
(Just to say it explicitly: this is def ok though! Pointing at upstream's main branch is something we do with some frequency.)